ogr_fdw_info.c

```diff
@@ -213,12 +257,35 @@ ogrGenerateSQL(const char* source, const char* layer)
 	}
 
 	if (! ogr_dr)
-	{
 		ogr_dr = GDALGetDatasetDriver(ogr_ds);
-	}
 
-	/* There should be a nicer way to do this */
-	strcpy(server_name, "myserver");
+	strcpy(server_name, server == NULL ? "myserver" : server);
+
+  if (options != NULL) {
```

Comment:
Astyle first?

Reply:
Fixed manually. I can astyle ... what options did you use?

Reply:
See `.astylerc`